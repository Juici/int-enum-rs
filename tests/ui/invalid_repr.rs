use int_enum::IntEnum;

#[derive(Clone, Copy, IntEnum)]
#[repr(align(1))]
pub enum InvalidRepr {
    Zero = 0,
    One = 1,
}

fn main() {}
