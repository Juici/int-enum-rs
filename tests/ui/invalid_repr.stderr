error: missing type in `repr` attribute
 --> tests/ui/invalid_repr.rs:3:1
  |
3 | #[repr(align(1))]
  | ^^^^^^^^^^^^^^^^^
4 | #[derive(Clone, Copy, IntEnum)]
  |                       ------- in this derive macro expansion
  |
  = note: valid reprs are `i8`, `u8`, `i16`, `u16`, `i32`, `u32`, `i64`, `u64`, `i128`, `u128`, `isize`, `usize`
  = note: this error originates in the derive macro `IntEnum` (in Nightly builds, run with -Z macro-backtrace for more info)
